ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-2.4.yaml

```diff
@@ -0,0 +1,47 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.4"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 16Gi
```

Comment:
I don't think we need 16G memory in operator job. what about 4Gi or 8Gi in the limits ?

Reply:
this is a copy of whatever was there initially. We can do a clean up in a follow up pr.